docs/release/k3s-release.md

```diff
@@ -0,0 +1,253 @@
+# K3S Kubernetes Patch Release Process
+This document details the K3S kubernetes patch release process.
+For reference, this document makes the following assumptions:
+
+```
+Current K3S Version: v1.22.11
+New K3S Version [New k8s patch]: v1.22.12
+
+Affected repos are:
+k3s-io/kubernetes
+k3s-io/k3s
+```
+
+# Before You Begin
+You’ll be primarily using git and go. Git can be installed via the local package manager. Make sure Go is installed and configured correctly, utilizing a “gopath”. This can be set via an environment variable called GOPATH. eg. export GOPATH=”${HOME}/go”, typically.
+
+## Clone and Setup Remotes
+Clone from upstream then add k3s-io fork and your personal fork.
+```sh
+# initial clone
+git clone --origin upstream \
+  https://github.com/kubernetes/kubernetes.git \
+  ${GOPATH}/src/github.com/kubernetes/kubernetes
+ 
+# cd into the repo
+cd ${GOPATH}/src/github.com/kubernetes/kubernetes
+ 
+# export your github name if different from user
+# add the k3s-io remote
+git remote add k3s-io https://github.com/k3s-io/kubernetes.git
+ 
+# fetch all remote branches and tags. If you receive a message saying that
+# previous tags will be "clobbered", add --force to the command below.
+git fetch --all --tags
+```
+# Rebasing and Generating Tags
+Establish a local branch for rebasing:
+
+```sh
+export GHUSER=<your-github-username>
```

Comment:
```suggestion
```
This is not needed, since we are not pushing to personal forks and not changing k3s mod to use it